src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs

```diff
@@ -1168,6 +1191,16 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
             };
         }
 
+        private string TrimTrailingSeparators(string s)
```

Comment:
Perhaps this helper should be placed in SourceFileResolver and reused between VB and C#?